src/store/unlocks/constants.js

```diff
@@ -1,20 +1,10 @@
-export const CURRENT_VERSION = 3;
+export const CURRENT_VERSION = 4;
```

Comment:
👏👏

Reply:
If you want to have this effect existing state, you'll need to add a version update function as well - like if you wanted to remove the default bosses from users' unlock lists